### ISSUE TITLE
Set up CI testing

### ISSUE THREAD

**Issue by sqs** [sourcegraph/webloop#1]
There's a .travis.yml file in the travis-ci branch, but it doesn't work yet. The difficulty is that Travis-CI uses Ubuntu 12.04 LTS VMs, which don't have an easy way of installing WebKitGTK+ 2.

See also: https://github.com/sourcegraph/go-webkit2/issues/1

**Comment by srgrn** [comment]
is this project still alive? most CI services are using docker now, it might be easier today than in 2014

**Comment by sqs** [comment]
Yep, this project is still alive. Indeed, it would be easier now.
